package.json

```diff
@@ -82,6 +82,6 @@
     ]
   },
   "engines": {
-    "node": "6.12.3 - 7.0.0 || >=8.9.4"
+    "node": ">=6.12.3 <8.0.0 || >=8.9.4 <10.0.0 || >=10.4.1"
```

Comment:
It might be good to add some tests to test this expression.

Reply:
Done.